pkgs/development/libraries/xgboost/default.nix

```diff
@@ -77,11 +85,7 @@ stdenv.mkDerivation rec {
     # https://github.com/dmlc/xgboost/blob/ccf43d4ba0a94e2f0a3cc5a526197539ae46f410/CMakeLists.txt#L145
     "-DCMAKE_C_COMPILER=${cudaPackages.backendStdenv.cc}/bin/gcc"
     "-DCMAKE_CXX_COMPILER=${cudaPackages.backendStdenv.cc}/bin/g++"
```

Comment:
I'd rather update these to refer to `effectiveStdenv`/`stdenv` instead of directly to `cudaPackages`

Reply:
Addressed in latest

Reply:
Oh let's not risk it with splicing... I think symbol hiding is actually safer

Reply:
Resolved in latest